kelk/env/src/context.rs

```diff
@@ -6,6 +6,9 @@ use alloc::vec::Vec;
 
 /// TODO
 pub trait ContextAPI {
+    /// TODO
+    fn msg_sender(&self) -> Result<Vec<u8>, KelkError>;
```

Comment:
This is not related to this PR. For all parameters we can use `get_param` function. Look here: https://github.com/zarbchain/kelk/blob/ebb329c48d09c5841be3b8997d837d221f98b63b/kelk/env/src/params.rs#L7